sdk/storage/azure-storage-common/storage_shared_key_credential.cpp

```diff
@@ -0,0 +1,20 @@
+namespace azure { namespace storage { namespace commons {
+
+class storage_shared_key_credential
+{
+public:
+
+//ctor
+    storage_shared_key_credential(std::string account_name, std::string account_key);
+
+    const std::string& get_account_name() const;
+    const std::string& get_account_key() const;
+    void set_account_key(std::string account_key);
+
+private:
+    std::mutex m_account_key_mutex;
+    std::string m_account_name;
```

Comment:
Missing `<string>` for `std::string`.

Reply:
(Also elsewhere many times)

Reply:
This is not a working version of the SDK that even compiles, but rather raising the interface that everyone agrees.

Reply:
I see, OK, feel free to disregard most such comments for now.